src/PlaywrightSharp.Tests/PageSetInputFilesTests.cs

```diff
@@ -55,11 +55,11 @@ public async Task ShouldSetFromMemory()
         {
             await Page.SetContentAsync("<input type=file>");
 
-            await Page.SetInputFilesAsync("input", new FilePayload
+            await Page.SetInputFilesAsync("input", new ElementHandleFiles
             {
                 Name = "test.txt",
                 MimeType = "text/plain",
-                Buffer = Convert.ToBase64String(Encoding.UTF8.GetBytes("this is a test"))
+                Buffer = Encoding.UTF8.GetBytes("this is a test"),

```

Comment:
This is a significant change, did you add it to the draft release/breaking changes yet?

Reply:
Yes Sir.